src/bindings.rs

```diff
@@ -584,12 +584,12 @@ pub enum IRRootParameterType {
 #[derive(Copy, Clone)]
 pub struct IRRootParameter {
     pub ParameterType: IRRootParameterType,
-    pub u_1: IRRootParameter__bindgen_ty_1,
+    pub u_1: IRRootParameter_u,
```

Comment:
Shouldn't the field be named `u` instead of `u_1`?

Reply:
We could add some more logic in the callback to get that result

Reply:
I think all I can do is `.anon_fields_prefix("u_")`, but I don't know how to get rid if the number at the end.

Reply:
Awh yeah I don't see it either in https://github.com/Traverse-Research/saxaboom/compare/main...switch-to-bindgen, guess it's not easy for now.